test/kpod_pull.bats

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bats
+
+load helpers
+
+IMAGE="alpine:latest"
+ROOT="$TESTDIR/crio"
+RUNROOT="$TESTDIR/crio-run"
+KPOD_OPTIONS="--root $ROOT --runroot $RUNROOT --storage-driver vfs"
```

Comment:
Might want to use `${STORAGE_OPTS}` instead of `--storage-driver vfs` like in other tests.

Reply:
Yes lets do this with ${STORAGE_OPTS},  I want to get rid of the hard coded --storage-driver vfs if at all possible.  @baude This means the tests will default to the default way that crio will run on a platform, which is better coverage then just running tests on vfs.

Reply:
fixed